native_client/ds_git_version.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/sh
 
 if [ `uname` = "Darwin" ]; then
-   export PATH="${TASKCLUSTER_TASK_DIR}/homebrew/opt/coreutils/libexec/gnubin:${PATH}"
-fi;
+   export PATH="/Users/build-user/TaskCluster/Workdir/tasks/tc-workdir/homebrew/opt/coreutils/libexec/gnubin:${PATH}"
```

Comment:
Very nasty, but Bazel resets the environment (on purpose), that was the quickest I could think of. Should not be a big big mes.